pkg/resources/cloudcontroller/digitalocean.go

```diff
@@ -0,0 +1,112 @@
+package cloudcontroller
+
+import (
+	"fmt"
+
+	"k8c.io/kubermatic/v2/pkg/resources"
+	"k8c.io/kubermatic/v2/pkg/resources/reconciling"
+	"k8c.io/kubermatic/v2/pkg/resources/vpnsidecar"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/pointer"
+)
+
+const (
+	DigitalOceanCCMDeploymentName = "digitalocean-cloud-controller-manager"
+	DigitalOceanVersion           = "0.1.39"
+)
+
+var (
+	digitalOceanResourceRequirements = corev1.ResourceRequirements{
+		Requests: corev1.ResourceList{
+			corev1.ResourceMemory: resource.MustParse("50Mi"),
+			corev1.ResourceCPU:    resource.MustParse("100m"),
+		},
+	}
+)
```

Comment:
Why do we need this?

Reply:
The resources were mentioned in [the DO-ccm yml](https://github.com/digitalocean/digitalocean-cloud-controller-manager/blob/8d20afd329c37057ed472db5f3dc4b5025f7eda6/releases/v0.1.39.yml#L43), I thought it was necessary